Documentation/ceph-csi-drivers.md

```diff
@@ -175,3 +175,68 @@ CSI-Addons supports the following operations:
   - [Annotating PersistentVolumeClaims](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.3.0/docs/reclaimspace.md#annotating-perstentvolumeclaims)
 - Network Fencing
   - [Creating a NetworkFence](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.3.0/docs/networkfence.md)
+
+### Enable RBD Encryption Support
+
+Ceph-CSI supports encrypting RBD volumes with LUKS encryption. More details can be found
+[here](https://github.com/ceph/ceph-csi/blob/v3.6.0/docs/deploy-rbd.md#encryption-for-rbd-volumes)
+with full list of supported encryption configurations. A sample configmap can be found
+[here](https://github.com/ceph/ceph-csi/blob/v3.6.0/examples/kms/vault/kms-config.yaml).
+
+Following steps demonstrate how to enable support for encryption:
+
+* Create the `rook-ceph-csi-kms-config` configmap with required encryption configuration in
+the same namespace where the Rook operator is deployed. An example is shown below:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rook-ceph-csi-kms-config
+  namespace: rook-ceph
+data:
+  config.json: |-
+    {
+      "user-secret-metadata": {
+        "encryptionKMSType": "metadata",
```

Comment:
Are there other encryption types we should mention?

Reply:
> Are there other encryption types we should mention?

This is just an example, other encryption types are explained here https://github.com/ceph/ceph-csi/blob/v3.6.0/docs/deploy-rbd.md#encryption-for-rbd-volumes 
which is linked in line 182

and full list of example at https://github.com/ceph/ceph-csi/blob/v3.6.0/examples/kms/vault/kms-config.yaml
linked in line 184 above